### ISSUE TITLE
conda not automatically initialized

### ISSUE THREAD

**Issue by KornelJahn** [olynch/scientific-fhs#2]
It seems that variable `condaInitScript` is not used by `fhsCommand` in [default.nix](https://github.com/olynch/scientific-fhs/blob/425cf6909a3e6e7a793d54681297fa9b30efc1d4/default.nix#L160). Therefore, the commands in `condaInitScript` need to be run manually at first use.

**Comment by olynch** [comment]
Yes, this is a feature not a bug; I want to explicitly control my conda environments. I'd consider adding this as an option if you send a PR.
